Rudder-Facebook/Classes/RudderFacebookIntegration.m

```diff
@@ -67,20 +67,23 @@ - (void) processRuderEvent: (nonnull RSMessage *) message {
             NSString *currency = [self extractCurrency:message.properties withKey:@"currency"];
             
             if (revenue) {
-                [FBSDKAppEvents logPurchase:[revenue doubleValue] currency:currency];
+                [FBSDKAppEvents.shared logPurchase:[revenue doubleValue] currency:currency];
                 
                 // Custom event
                 NSMutableDictionary *properties = [message.properties mutableCopy];
                 [properties setObject:currency forKey:FBSDKAppEventParameterNameCurrency];
-                [FBSDKAppEvents logEvent:truncatedEvent
+                [FBSDKAppEvents.shared logEvent:truncatedEvent
                               valueToSum:[revenue doubleValue]
                               parameters:properties];
                 
             }
             else {
-                [FBSDKAppEvents logEvent:truncatedEvent
+                [FBSDKAppEvents.shared logEvent:truncatedEvent
                               parameters:message.properties];
             }
+            [FBSDKAppEvents.shared logEvent: FBSDKAppEventNameAddedToCart
+                    valueToSum: 100.0
+                    parameters: message.properties];
```

Comment:
Why are we doing this ?

Reply:
I'll remove this. This need not to be present.